generators/app/templates/gulp/_injecting.js

```diff
@@ -33,7 +33,7 @@ gulp.task('inject-all', ['styles', 'wiredep', 'bower-fonts', 'environment', 'bui
 });
 
 // build styles to tmp
-gulp.task('styles', ['clean'], function () {
+gulp.task('styles', ['clean-styles'], function () {
```

Comment:
gulp tasks: please leave the original gulp tasks in tact, I assume that some of the gulp tasks break because of these changes.